src/encoded/static/components/search.js

```diff
@@ -508,6 +509,53 @@ const Target = auditDecor(TargetComponent);
 globals.listingViews.register(Target, 'Target');
 
 
+/* eslint-disable react/prefer-stateless-function */
+class GeneComponent extends React.Component {
+    render() {
+        const result = this.props.context;
+        return (
+            <li>
+                <div className="clearfix">
+                    <PickerActions {...this.props} />
+                    <div className="pull-right search-meta">
+                        <p className="type meta-title">Gene</p>
+                        {this.props.auditIndicators(result.audit, result['@id'], { session: this.context.session, search: true })}
+                    </div>
+                    <div className="accession">
+                        <a href={result['@id']}>
+                            {result.symbol}
+                            {result.organism && result.organism.scientific_name ? <em>{` (${result.organism.scientific_name})`}</em> : null}
+                        </a>
+                    </div>
+                    <div className="data-row">
+                        <strong>External resources: </strong>
+                        {result.dbxrefs && result.dbxrefs.length ?
+                            <DbxrefList context={result} dbxrefs={result.dbxrefs} />
+                        : <em>None submitted</em> }
+                    </div>
+                </div>
+                {this.props.auditDetail(result.audit, result['@id'], { session: this.context.session, except: result['@id'], forcedEditLink: true })}
+            </li>
+        );
+    }
+}
+/* eslint-enable react/prefer-stateless-function */
+
+GeneComponent.propTypes = {
+    context: PropTypes.object.isRequired, // Target search results
+    auditIndicators: PropTypes.func.isRequired, // Audit decorator function
+    auditDetail: PropTypes.func.isRequired, // Audit decorator function
+};
+
+GeneComponent.contextTypes = {
+    session: PropTypes.object, // Login information from <App>
+};
+
+const Gene = auditDecor(GeneComponent);
+
+globals.listingViews.register(Gene, 'Gene');
```

Comment:
We’re moving towards listingViews being in the modules they represent, so in this case `GeneComponent` would be in gene.js and called `ListingComponent` if you were to be consistent with other modules (you can look at genetic_modification.js for just one example). If you prefer to leave it here, then don’t worry about it, but moving listingViews to their own modules has been the way we’ve been going, just so that search.js doesn’t keep growing and possibly needing to import parts specific to certain object types.

Also, like above, you should remove the eslint-disable comments and make this a stateless component.